extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -377,6 +384,18 @@ export class AmpConsent extends AMP.BaseElement {
     const config = parseJson(script.textContent);
     const consents = config['consents'];
     user().assert(consents, `${TAG}: consents config is required`);
+
+    if (!this.isSupportMulti_) {
+      // Assert single consent instance
+      user().assert(Object.keys(consents).length <= 1,
```

Comment:
I thought we we just going to honor the first one? Not break?

Reply:
Right I was going to just honor the first one. But then I realize it is always safer to break. For example, it is always better to throw error if someone ask for three consents to unblock content, rather than unblock content without dealing with the other two.